editor/test/integration/test_util.clj

```diff
@@ -277,7 +277,8 @@
      [workspace project app-view])))
 
 (defn- load-system-and-project-raw [path]
-  (test-support/with-clean-system
+  (test-support/with-clean-system {:cache-size 1000
+                                   :cache-retain? project/cache-retain?}
```

Comment:
Tests default to a cache size of `1000`, just like before. Increasing this doesn't appear to make things faster.